D/DynarePreprocessor/build_tarballs.jl

```diff
@@ -0,0 +1,40 @@
+using BinaryBuilder
+
+
+name = "DynarePreprocessor"
+version = v"4.8.0-5"
```

Comment:
I don't think this will work, this is a prerelease version in semver, which is not allowed. If this is the actual upstream version, you may need to find a mapping of the versions used by upstream to versions accepted bu Julia. We do that eg in Singular_jll